.github/workflows/verify.yml

```diff
@@ -144,24 +164,47 @@ jobs:
         with:
           files: "package.json"
 
+      - name: Check for bun.lockb existence
+        id: bun_lock_check
+        uses: andstor/file-existence-action@v2
+        with:
+          files: "bun.lockb"
+
+      - name: Check if Sorbet is in use
+        id: sorbet_check
+        uses: andstor/file-existence-action@v2
+        with:
+          files: "sorbet"
+
       - name: Set up Ruby and install gems
         uses: ruby/setup-ruby@v1
         with:
           bundler-cache: true
 
       - name: Set up Node
         uses: actions/setup-node@v3
-        if: steps.package_json_check.outputs.files_exists == 'true' && hashFiles('yarn.lock') != ''
+        if: steps.package_json_check.outputs.files_exists == 'true' && steps.bun_lock_check.outputs.files_exists == 'false' && hashFiles('yarn.lock') != ''
         with:
           node-version: ${{ inputs.node-version }}
           node-version-file: ${{ inputs.node-version-file }}
           cache: "yarn"
 
       - name: Install Node packages
-        if: steps.package_json_check.outputs.files_exists == 'true'
+        if: steps.package_json_check.outputs.files_exists == 'true' && steps.bun_lock_check.outputs.files_exists == 'false'
         run: |
           yarn install --pure-lockfile
 
+      - name: Set up Bun
```

Comment:
Once this is merged, I think I'll restructure things so there is just a `Set up JavaScript` step which handles Yarn or Bun conditionally. Then there won't be so many skipped steps listed in the GitHub Actions output.